apps/nextjs-website/src/components/Video.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+import { Video as VideoRC } from '@react-components/components';
+import { VideoImageProps } from '@react-components/types';
+import { VideoSection } from '@/lib/fetch/types/PageSection';
+
+const makeVideoProps = ({
+  title,
+  subtitle,
+  src,
+  fallback,
+  playButtonLabel,
+  isCentered,
+  ...rest
+}: VideoSection): VideoImageProps => ({
+  ...rest,
+  title,
+  subtitle,
+  ...(src.data && {
+    src: {
+      url: src.data.attributes.url,
+      alternativeText: src.data.attributes.alternativeText || null,
```

Comment:
```suggestion
      alternativeText: src.data.attributes.alternativeText,
```
`alternativeText` should already be `string | null`